openid-sharedsignals-framework-1_0.md

```diff
@@ -175,20 +164,12 @@ normative:
     target: https://openid.net/wg/resources/naming-and-contents-of-specifications/
     title: OpenID Naming and Content of Specifications
 
-informative:
-  USECASES:
```

Comment:
this informative reference was never used... so i removed it. happy to add it back if we need it, just the `make` command was giving a warning.